hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/protocolPB/OzoneManagerRequestHandler.java

```diff
@@ -917,6 +917,33 @@ public static OMResponse disallowGetFileStatusWithECReplicationConfig(
     return resp;
   }
 
+  @RequestFeatureValidator(
+          conditions = ValidationCondition.OLDER_CLIENT_REQUESTS,
+          processingPhase = RequestProcessingPhase.POST_PROCESS,
+          requestType = Type.EchoRPC
+  )
+  public static OMResponse echoRPC(
+          OMRequest req, OMResponse resp, ValidationContext ctx)
+          throws ServiceException {
+
+    if (!resp.hasEchoRPCResponse()) {
+      return resp;
+    }
+    if (!req.getEchoRPCRequest().hasIsEmptyResp() ||
```

Comment:
Can we support the case where the request has no payload but the response does?

Reply:
Make sense, I'll change this part accordingly!

Reply:
Hey Ritesh, I add an [option](https://github.com/apache/ozone/pull/3614/files#diff-c2423ca69889970743c3fc1e72385459675b8ac9dd4d80227ea4b92090625690R60
) to specify response size, and then generate the random byte array when OM prepare its [response](https://github.com/apache/ozone/pull/3614/files#diff-4e1ce03634e167877ec3c4d055a744b753d087ef7d9a3ed842f9936ef519aa71R937). Feel free to let me know how you think!